pkg/telemetry/telemetry.go

```diff
@@ -86,6 +87,11 @@ func Start(ctx context.Context, honeycomb_api_key string, honeycomb_dataset stri
 	AddStringSpanAttribute(span, "uuid", machineUUID)
 	AddStringSpanAttribute(span, "osinfo/os", runtime.GOOS)
 	AddStringSpanAttribute(span, "osinfo/arch", runtime.GOARCH)
+	runningInCi := os.Getenv("CI")
```

Comment:
We want to make sure to munge this to lower just to be sure we're not caught out by different implementations

Reply:
Done